src/ovirt-engine-hyper-upgrade

```diff
@@ -358,18 +393,18 @@ def main():
     logger = logging.getLogger(__name__)
 
     # Dictionary that keep the release data like:
-    # version, repo_to_enable, repo_to_disable
+    # version, the difference between repo_to_enable and repo_to_disable
     check_releases = {}
     check_releases['repos'] = [
```

Comment:
I think you should move this too to top-level and reuse.

Ideally, imo, the versions strings - '4.0', '4.1' - should only appear here. All the rest of the code should work with any version and do not care.

I also think we should remove the specific options '--upgrade-to-rhv-4-1', '--upgrade-to-rhv-4-2'. We should by default upgrade to latest (based on the data in this list), and allow a single option '--upgrade-to=<ver>', where <ver> is also checked against 'version' in this list.